### PR TITLE
Another-commit

### PR DESCRIPTION
**Pull request by Aryan-Fexle-May2022**
Commited new Classes-Pattern Programs

### CODE REVIEW

File: src/package.xml [hunk 1]
```diff
@@ -6,6 +6,13 @@
         <members>StudentManager</members>
 		<members>PyramidProgram</members>
         <members>StarPattern2</members>
```
Comment (MittalPriyanka) [inline src/package.xml:8]: Fix Indentation

File: src/classes/RightUpDownTriangle.cls [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                for(Integer j=1;j<=i-n/2-1;j++) {
+                    s = s + '_';
+                }
+                for(Integer j=1;j<=a/2;j++) {
+                    s = s + '*';
+                }
+                a = a - 2;
+            }
+            System.debug(s);
+
```
Comment (MittalPriyanka) [inline src/classes/RightUpDownTriangle.cls]: Line break is not required here.

File: src/package.xml (hunk 1 above)
Comment (Aryan-Fexle-May2022) [reply src/package.xml:8]: Done.

File: src/classes/RightUpDownTriangle.cls (hunk 2 above)
Comment (Aryan-Fexle-May2022) [reply src/classes/RightUpDownTriangle.cls]: Done.

File: src/classes/PrintAlphabetA.cls [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                
+                if(j>=k && j<=t && m )
+                {
+                    s = s + 'x';  
+                }
+                else
+                {
+                    s = s + '_';
+                }
+
```
Comment (MittalPriyanka) [inline src/classes/PrintAlphabetA.cls]: remove extra line break.
Review all classes again!

Comment (Aryan-Fexle-May2022) [reply src/classes/PrintAlphabetA.cls]: done

**Review by MittalPriyanka — APPROVED** [review]

**Pull request merged**